UPGRADE-8.0.md

```diff
@@ -0,0 +1,18 @@
+UPGRADE FROM 7.4 to 8.0
+=======================
+
+Symfony 7.4 and Symfony 8.0 are released simultaneously at the end of November 2025. According to the Symfony
+release process, both versions have the same features, but Symfony 8.0 doesn't include any deprecated features.
+To upgrade, make sure to resolve all deprecation notices.
+Read more about this in the [Symfony documentation](https://symfony.com/doc/8.0/setup/upgrade_major.html).
+
+Table of Contents
```

Comment:
same as a few months ago : the toc is unmaintainable during the dev period, it should be added after/just before the release
7.3 is missing one BTW ;)

Reply:
Ah yes I forgot about it! I removed it and I'm having a look at 7.3